dumper/app/src/main/java/com/google/edwmigration/dumper/application/dumper/ConnectorArguments.java

```diff
@@ -416,7 +416,7 @@ public List<String> getSchemata() {
     }
 
     public boolean isAssessment() {
-        return BooleanUtils.isTrue(getOptions().valueOf(optionAssessment));
+        return getOptions().has(optionAssessment);
```

Comment:
I think `has` just checks, whether the option is there, not whether the boolean has value `true`.
https://github.com/jopt-simple/jopt-simple/blob/master/net.sf.joptsimple/src/main/java/joptsimple/OptionSet.java#L94

I would suggest:
```
return getOptions().valueOf(optionAssessment);
```

Reply:
Apparently, none of this is enough. Argument parser is interesting, allowing multiple values combination and evaluating options lazely.

I tried to go with:
```
public boolean isAssessment() {
        OptionSet options = getOptions();
        if (options.has(optionAssessment)) {
            return options.hasArgument(optionAssessment) ? options.valueOf(optionAssessment) : true;
        }
        return false;
    }
```

But it also doesn't work as expected, correctly evaluating ` `, `--assessment`, `--assessment [no,false,0]`, but accepting everything else as true, without failing.

I decided to make the flag to expect required argument. It doesn't allows to use `--assessment` as is, but at least it fails for something like `--assessment folse`.